### PR TITLE
OU filter for customizing of stats and reports

### PR DESCRIPTION
**Pull request by joroca**

### CODE REVIEW

File: gecoscc/templates/report.jinja2 [hunk 1]
```diff
@@ -73,11 +73,11 @@
       integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
       crossorigin="anonymous"></script>
     <!-- TODO: Sustituir CDN -->
-    <link rel="stylesheet" type="text/css" href="http://cdn.datatables.net/1.10.19/css/jquery.dataTables.min.css" />
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.19/css/jquery.dataTables.min.css" />
```
Comment (alfem) [inline gecoscc/templates/report.jinja2]: Please use local resources instead of cdn urls.

File: gecoscc/views/report_printers.py [hunk 2]
```diff
@@ -64,34 +65,32 @@ def report_printers(context, request, file_ext):
...
-        ou_id = request.GET.get('ou_id', None)
-        if ou_id is None:
-            raise HTTPBadRequest()
-        
-        ou_visibles = request.user.get('ou_managed', []) + request.user.get('ou_readonly', [])
-        for oid in ou_visibles:
-            if oid == ou_id:
-                ou = ou_id
+
+    # Get managed ous
```
Comment (alfem) [inline gecoscc/views/report_printers.py]: This piece of code is repeated four times. Consider to move it to a common function/method

Comment (joroca) [reply gecoscc/views/report_printers.py]: Done.

File: gecoscc/templates/report.jinja2 (hunk 1 above)
Comment (joroca) [reply gecoscc/templates/report.jinja2]: Done.

**Pull request merged**